.ocamlformat

```diff
@@ -1 +1 @@
-version = 0.26.0
+version = 0.26.2
```

Comment:
Is this change on purpose or because you had this version locally in your switch? IOW, should it be commited?

Reply:
You're right, reverted in https://github.com/reasonml/reason-react/pull/852/commits/e5eeb3f23fd08fe028f7220f87caa5b86d3161c2.